src/Azure/AzureAD/Authentication.AzureADB2C.UI/src/AzureAdB2COptions.cs

```diff
@@ -9,6 +10,7 @@ namespace Microsoft.AspNetCore.Authentication.AzureADB2C.UI
     /// <summary>
     /// Options for configuring authentication using Azure Active Directory B2C.
     /// </summary>
+    [Obsolete("This is obsolete and will be removed in a future version. Use Microsoft.Identity.Web instead. See https://aka.ms/ms-identity-web.")]
```

Comment:
If we are obsoleting this APIs, we need to point to a URL within the official docs that contains details on the migration process.

Reply:
@jmprieur can you write something like that? Also, for breaking changes, we usually write an announcement like https://github.com/aspnet/Announcements/issues/414. Can you create something like that so we can link to it?